packages/flutter_tools/lib/src/dart_pub_json_formatter.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:collection';
+
+class Package {
```

Comment:
this definitely needs a dartdoc, and maybe a more specific name. Note that since the convention in Dart is to just do `import 'foo.dart';`, every name gets imported into that library's namespace